python/SKFlat.py

```diff
@@ -356,8 +321,8 @@
   fi
 
   echo "#### running ####"
-  echo "root -l -b -q run_${{SECTION}}.C"
-  root -l -b -q run_${{SECTION}}.C 2> err.log
+  echo "root -l -b -q {1}/run_${{SECTION}}.C"
+  root -l -b -q {1}/run_${{SECTION}}.C 2> err.log
```

Comment:
It is a question, rather than a comment.
when I tried **echo "${{SECTION}}"** in my test script.
It throws an error: **bash: ${{SECTION}}: bad substitution**
I wonder if this bash command is working.

Reply:
The output will be ${SECTION} not ${{SECTION}}, so it will work. '{{' is used because it is inside ''' ~~~ '''.format()